iMessageClone/iMessageChatMessageComposerView.swift

```diff
@@ -10,15 +10,15 @@ final class iMessageChatMessageComposerView: ChatMessageComposerView {
     override func setUpLayout() {
         super.setUpLayout()
 
-        centerLeftContainer.removeArrangedSubview(commandsButton)
+        centerLeftContainer.removeArrangedSubview(commandButton)
         centerRightContainer.removeArrangedSubview(sendButton)
         centerContainer.removeArrangedSubview(centerRightContainer)
 
-        centerContentContainer.axis = .horizontal
-        centerContentContainer.distribution = .natural
-        centerContentContainer.alignment = .fill
-        centerContentContainer.subviews.forEach {
-            centerContentContainer.removeArrangedSubview($0)
+        centerInputContainer.axis = .horizontal
+        centerInputContainer.distribution = .natural
+        centerInputContainer.alignment = .fill
+        centerInputContainer.subviews.forEach {
+            centerInputContainer.removeArrangedSubview($0)
         }
```

Comment:
We have `removeAllArrangedSubviews()` method on `ContainerStackView` (it was you who added it 😅 😁 )

Reply:
Yes this is old code, when I introduced that new method, forgot to change it here 😅 These are changes done by the name refactoring on Xcode